<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1378</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" columnstretch="7,0,2">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>900</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>190</red>
              <green>244</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>190</red>
              <green>244</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNoteButton">
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>      Добавить заметку</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/new/icons1/img/free-icon-font-edit-3917361.png</normaloff>:/new/icons1/img/free-icon-font-edit-3917361.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string notr="true" comment="Начать запись показаний">Начать запись показаний</string>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string>начать измерения</string>
          </property>
          <property name="accessibleName">
           <string>екеке</string>
          </property>
          <property name="accessibleDescription">
           <string>нннен</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-play-alt-3914307.png</normaloff>:/new/icons1/img/free-icon-font-play-alt-3914307.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="Остановить и сохранить файл">Остановить и сохранить файл</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-stop-3917685.png</normaloff>:/new/icons1/img/free-icon-font-stop-3917685.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,4,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="newFileButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="toolTip">
             <string comment="Создать файл для сохранения измерений">Создать файл для сохранения измерений</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/new/icons1/img/free-icon-font-add-document-3914213.png</normaloff>:/new/icons1/img/free-icon-font-add-document-3914213.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>9</red>
                  <green>190</green>
                  <blue>93</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>9</red>
                  <green>190</green>
                  <blue>93</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>9</red>
                  <green>190</green>
                  <blue>93</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="toolTip">
             <string comment="Экспорт графика в изображение ">Экспорт графика в изображение </string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/new/icons1/img/free-icon-font-jpg-3914265.png</normaloff>:/new/icons1/img/free-icon-font-jpg-3914265.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshCom">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>35</height>
             </size>
            </property>
            <property name="toolTip">
             <string comment="Обновить список портов">Обновить список портов</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/new/icons1/img/free-icon-font-refresh-3917293.png</normaloff>:/new/icons1/img/free-icon-font-refresh-3917293.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="portStatusLabel">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timerStatus">
            <property name="font">
             <font>
              <family>Segoe UI Light</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Создайте файл таблицы. После нажатия на кнопку 
&quot;старт&quot; в нем будут записаны показания датчиков</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="widget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,0,3,0,3,0,3,0">
        <item row="3" column="3">
         <widget class="QCheckBox" name="checkGraph2">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string comment="Отобразить на графике">Отобразить на графике</string>
          </property>
          <property name="text">
           <string>График</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="Value2">
          <property name="font">
           <font>
            <family>Swis721 WGL4 BT</family>
            <pointsize>33</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QComboBox" name="combo4">
          <property name="toolTip">
           <string comment="Список датчиков">Список датчиков</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="Value1">
          <property name="font">
           <font>
            <family>Swis721 WGL4 BT</family>
            <pointsize>33</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QComboBox" name="combo3">
          <property name="toolTip">
           <string comment="Список датчиков">Список датчиков</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLabel" name="Value4">
          <property name="font">
           <font>
            <family>Swis721 WGL4 BT</family>
            <pointsize>33</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="Value3">
          <property name="font">
           <font>
            <family>Swis721 WGL4 BT</family>
            <pointsize>33</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="combo2">
          <property name="toolTip">
           <string comment="Список датчиков">Список датчиков</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo1">
          <property name="toolTip">
           <string comment="Список датчиков">Список датчиков</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="9">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QCheckBox" name="checkGraph3">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string comment="Отобразить на графике">Отобразить на графике</string>
          </property>
          <property name="text">
           <string>График</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkGraph1">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string comment="Отобразить на графике">Отобразить на графике</string>
          </property>
          <property name="text">
           <string>График</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QCheckBox" name="checkGraph4">
          <property name="font">
           <font>
            <family>Segoe UI Light</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string comment="Отобразить на графике">Отобразить на графике</string>
          </property>
          <property name="text">
           <string>График</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</normaloff>:/new/icons1/img/free-icon-font-stats-3916729.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="6">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_13">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1378</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
